content/03.methods.md

```diff
@@ -83,19 +83,19 @@ We used GATK Haplotype caller.
+The input interval BED files for both panel and WXS data provided by the manufacturers were padded by 100 bp on each side for all variant calling algorithm runs.
+For WGS calling, we utilized the non-padded BROAD Institute interval calling list [`wgs_calling_regions.hg38.interval_list`](https://console.cloud.google.com/storage/browser/_details/genomics-public-data/resources/broad/hg38/v0/wgs_calling_regions.hg38.interval_list), comprised of the full genome minus N bases, unless otherwise noted below. 
+Strelka2 [@doi:10/gdwrp4] v2.9.3 was run using default parameters for canonical chromosomes (chr1-22, X,Y,M), as recommended by the authors.
 The final Strelka2 VCF was filtered for PASS variants.
 Mutect2 from GATK v4.1.1.0 was run following Broad best practices outlined from their Workflow Description Language (WDL) [@url:https://github.com/broadinstitute/gatk/blob/4.1.1.0/scripts/mutect2_wdl/mutect2.wdl].
 The final Mutect2 VCF was filtered for PASS variants. 
-Lancet [@doi:10.1038/s42003-018-0023-9] v1.0.7 [@url:https://github.com/nygenome/lancet] was run using default parameters, unless noted below.
-For input intervals to Lancet, a reference BED was created by using only the UTR, exome, and start/stop codon features of the GENCODE 31 reference.  
-Per recommendations by the New York Genome Center, the Lancet input intervals were augmented with PASS variant calls from Strelka2 and Mutect2 as validation.
-VarDictJava [@doi:10.1093/nar/gkw227] v1.58 [@url:https://github.com/AstraZeneca-NGS/VarDictJava] was run using the hg38 fasta reference with the same BED intervals used for Mutect2.  
-Parameters and filtering followed BCBIO standards except that variants with a variant allele frequency (VAF) >= 0.05 (instead of >= 0.10) were retained. 
+To manage memory issues, VarDictJava [@doi:10.1093/nar/gkw227] v1.58 [@url:https://github.com/AstraZeneca-NGS/VarDictJava] was run using 20Kb interval chunks of the input BED, padded by 100 bp on each side, such that if an INDEL occurred in between intervals, it would be captured.
+Parameters and filtering followed [BCBIO standards](https://github.com/bcbio/bcbio-nextgen) except that variants with a variant allele frequency (VAF) >= 0.05 (instead of >= 0.10) were retained. 
 The 0.05 VAF increased the true positive rate for INDELs and decreased the false positive rate for SNVs when using VarDict in consensus calling. 
 The final VCF was filtered for PASS variants with TYPE=StronglySomatic.
+Lancet [@doi:10.1038/s42003-018-0023-9] v1.0.7 [@url:https://github.com/nygenome/lancet] was run using default parameters, unless noted below.
+For input intervals to Lancet, a reference BED was created by using only the UTR, exome, and start/stop codon features of the GENCODE 31 reference and padded by 300 bp on each side. 
+Per recommendations by the New York Genome Center [@doi:10.1101/623702], the Lancet input intervals described above were augmented with PASS variant calls from Strelka2 and Mutect2 as validation.
```

Comment:
Maybe something like?
```suggestion
Per recommendations by the New York Genome Center [@doi:10.1101/623702], the Lancet input intervals described above were augmented with padded regions surrounding PASS variant calls from Strelka2 and Mutect2 as validation.
```

Reply:
I feel like this sounds like the variant calls from strelka/mutect were padded, but it was that the same files as above were augmented with strelka2/mutect2 calls, if not present already...

Reply:
My understanding was that the variant calls *are* padded... That is if there was a call from strelka or mutect at base 500, the region 200-800 would be added to the lancet intervals (if it wasn't already in there).

Reply:
hmm, let me get clarification from @migbro - wasn't my understanding but I didn't run it :)

Reply:
ok... so I got some clarification: 
`for lancet, incoming regions consist of the CDS enhanced with variant calls not in the CDS from strelka2 and mutect2. just like in vardict, they will be padded by the software by 300bp, UNLIKE vardict, since it’s not really WGS, probably not a whole lot of overlap between intervals, so, yeah, in going throught this though exercise, the total bp length can vary a lot sample-to-sample` - i think we should reword as this is run with padding, input intervals not padded.

Reply:
and perhaps, TMB should only be calculated using mutect2/strelka bc they are not padded

Reply:
> and perhaps, TMB should only be calculated using mutect2/strelka bc they are not padded

Agreed, though I think the main concern would be is less about padding than about the fact that every sample has its own input interval.

How about something like this for the lines in question:

```
For input intervals to Lancet, a reference BED was created by using only the UTR, exome, and start/stop codon features of the GENCODE 31 reference, augmented as recommended with PASS variant calls from Strelka2 and Mutect2 [@doi:10.1101/623702].
These intervals were then padded by 300 bp on each side during Lancet variant calling.

Reply:
right, yes that is what i was getting at. sounds better, will update.